src/window_help.h

```diff
@@ -49,13 +49,25 @@ class Window_Help : public Window_Base {
 	 */
 	void Clear();
 
+	/**
+	 * Adds text to the help window. This does not overwrite the old content.
+	 *
+	 * @param text text to add.
+	 * @param color text color.
+	 * @param align text alignment.
+	 * @param halfwidthspace if half width spaces should be used.
+	 */
+	void AddText(std::string text, int color = Font::ColorDefault, Text::Alignment align = Text::AlignLeft, bool halfwidthspace = true);
+
 private:
 	/** Text to draw. */
 	std::string text;
 	/** Color of Text to draw. */
 	int color = Font::ColorDefault;
 	/** Alignment of text. */
 	Text::Alignment align;
+	/** Current text position */
+	int x = 0;
```

Comment:
this shadows (same name) the x from the Window class which indicates the x position of the window on screen.
Can you rename it to something else like ``text_x_offset`` ?

Reply:
Fixed.